docs/index.md

```diff
@@ -19,10 +19,10 @@ PM > Install-Package Arcus.Messaging.* -Version 0.1.0
 # Features
 
 - Support for using message pumps for the following brokers:
-    - Azure Service Bus ([docs](features/message-pumps/service-bus))
-    - Azure Service Bus Extensions ([docs](features/service-bus))
+    - Azure Service Bus ([docs](features/message-pumps/service-bus) | [extensions](features/service-bus))
 - Support for exposing TCP health probes to periodically check liveness/readiness of the host ([docs](features/tcp-health-probe))
 - Customize message pumps ([docs](features/message-pumps/customization))
+    - Fallback message handlers
 
 ## Older versions
```

Comment:
Here you can now add the v0.2 as 'older version' since we released v0.3.

Reply:
Thanks for the heads up!